dasharo-security/tpm-support.robot

```diff
@@ -121,7 +114,7 @@ TPM003.002 Check TPM Physical Presence Interface (Ubuntu)
     Login To Linux
     Switch To Root User
     ${out}=    Execute Command In Terminal    cat /sys/class/tpm/tpm0/ppi/version
-    Should Contain    ${out}    1.3
+    Should Contain Any    ${out}    1.2    1.3
```

Comment:
I don't think it should be any.

If the expected TPM version is 2, then PPI version is 1.3. For TPM1.2 the PPI will be 1.2